code/mod_table/mod_table.cpp

```diff
@@ -58,6 +58,11 @@ std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_p2;
 std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_s1;
 bool Use_engine_wash_intensity;
 bool Ai_before_physics;
+SCP_vector<int> Required_render_ext;
+
+SCP_vector<std::pair<SCP_string, int>> req_render_ext_pairs = {
+	std::make_pair("BPTC", static_cast<int>(CAPABILITY_BPTC))
```

Comment:
Maybe call this "BPTC Texture Compression" to make it more explicit what it means.